src/style.css

```
@@ -0,0 +1,69 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  --header-height: 60px;
+  --blue-color: #00acdc;
+}
+
+.header {
+  /* width: 100%; */
+  height: var(--header-height);
+  display: flex;
+  justify-content: flex-end;
```

```suggestion
  justify-content: space-between;
```
Instead of giving your logo `position: absolute` it's better to use here space-between to make space between logo and nav